scrapers/covid-19/govScrapers/getMalaysia.js

```diff
@@ -0,0 +1,44 @@
+const nightmare = require('nightmare');
+const browser = nightmare({ show: false });
+const cheerio = require('cheerio');
+const logger = require('../../../utils/logger');
+const columns = ['state', 'cases', 'newCases', 'deaths', 'population'];
+
+const mapRows = (_, row) => {
+	const state = { updated: Date.now() };
+	cheerio(row).children('div').each((index, cell) => {
+		cell = cheerio.load(cell);
+		switch (index) {
+			case 0:
+				state[columns[index]] = cell.text().trim();
+				break;
+			case 4:
+				// eslint-disable-next-line no-case-declarations
+				const arr = cell.text().trim().replace(',', '').split('.');
+				state[columns[index]] = arr[1] ? parseInt(arr[0] + (parseInt(arr[1]) * 1000)) : parseInt(arr[0]) * 1000;
+				break;
+			default:
+				state[columns[index]] = parseInt(cell.text().trim().replace(',', ''));
+				break;
+		}
+	});
+	return state;
+};
+
+const malaysiaData = async () => {
+	try {
+		const html = cheerio.load(await browser
+			.goto('https://datastudio.google.com/u/0/reporting/cfca4946-cfc8-4fb4-804f-bb58aa6c35ca/page/1M%27')
```

Comment:
this URL is based from this [Malaysia gov](https://www.dosm.gov.my/v1/index.php?r=column/cone&menu_id=UjJoNk9OalhZWlVHdExiaGF1OW13UT09)